psydac/feec/tests/test_commuting_projections.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: UTF-8 -*-
```

Comment:
I think that we should also add tests in 1D and 2D.

Reply:
done